src/coreclr/jit/instrsxarch.h

```diff
@@ -621,6 +621,17 @@ INST3(lzcnt,            "lzcnt",            IUM_WR, BAD_CODE,     BAD_CODE,
 // POPCNT
 INST3(popcnt,           "popcnt",           IUM_WR, BAD_CODE,     BAD_CODE,     SSEFLT(0xB8),                            Resets_OF      | Resets_SF     | Writes_ZF     | Resets_AF     | Resets_PF     | Resets_CF )
 
+// SHA1
+INST3(sha1rnds4,        "sha1rnds4",       IUM_WR, BAD_CODE,     BAD_CODE,  0xcc,                                        Undefined_OF   | Undefined_SF  | Writes_ZF     | Undefined_AF  | Undefined_PF  | Writes_CF )
+INST2(sha1nexte,        "sha1nexte",       IUM_WR, BAD_CODE,     0xc8,      Undefined_OF   | Undefined_SF  | Writes_ZF     | Undefined_AF  | Undefined_PF  | Writes_CF )
```

Comment:
Just curious: how did it get that `0xc8` for `sha1nexte` for example?

Reply:
> Just curious: how did it get that `0xc8` for `sha1nexte` for example?

I looked at the opcode table in the kernel :D
https://github.com/torvalds/linux/blob/c9e6606c7fe92b50a02ce51dda82586ebdf99b48/arch/x86/lib/x86-opcode-map.txt#L790